### DISCUSSION TITLE
come creare il registro elettronico

### DISCUSSION THREAD

**Discussion by CobraOfficial01** [iisgiua/giuaschool#481] [Richieste di aiuto]
salve come si crea il registro elettronico, è possibile ricevere aiuto?

**Comment by trinko** [comment]
Vedi la documentazione su l'installazione:

- [https://iisgiua.github.io/giuaschool-docs/install.html](https://iisgiua.github.io/giuaschool-docs/install.html)

**Reply by CobraOfficial01** [reply]
ok ma dopo come procedo?

**Reply by trinko** [reply]
Dopo l'installazione devi procedere con la configurazione e l'importazione degli utenti:
- [Configurazione](https://iisgiua.github.io/giuaschool-docs/configure.html)

**Reply by trinko** [reply]
In alternativa, se ti serve solo una installazione di test, ti consiglio di installare l'ambiente docker e caricare l'immagine **giuaschool-test** che contiene i principali dati fittizi.
Per l'installazione tramite docker, vedi:
- [Installazione tramite docker](https://iisgiua.github.io/giuaschool-docs/install-docker.html)

**Reply by CobraOfficial01** [reply]
salve da dove si scarica l'applicazione?

**Reply by CobraOfficial01** [reply]
salve non sono riuscito ad installare il registro
